airflow/testfiles/airflow_settings.yaml

```diff
@@ -0,0 +1,23 @@
+# This file allows you to configure Airflow Connections, Pools, and Variables in a single place for local development only.
+# NOTE: If putting a dict in conn_extra, please wrap in single quotes.
+
+# For more information, refer to our docs: https://www.astronomer.io/docs/cloud/stable/develop/customize-image#configure-airflowsettingsyaml
+# For issues or questions, reach out to support@astronomer.io
```

Comment:
Can we also add a link to the repo issues so they can create tickets directly in the repo